201-decrypt-running-windows-vm/azuredeploy.json

```diff
@@ -32,7 +32,7 @@
     "extensionName": "AzureDiskEncryption",
     "extensionVersion": "1.1",
     "encryptionOperation": "DisableEncryption",
-    "updateEncryptionSettingsUrl": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-',parameters('volumeType'),'.json')]",	
+    "updateEncryptionSettingsUrl": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-windows-vm/updateEncryptionSettings-',parameters('volumeType'),'.json')]"
```

Comment:
add a parameter for _artifactsLocation and _artifactsLocationSasToken according to https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc .
Then in a variable, concat to something similar as:
"url": "[concat(parameters('_artifactsLocation'), '/nestedtemplates/templatename.json', parameters('_artifactsLocationSasToken'))]"